scripts/translations.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+languages = [
+    ('Simplified Chinese'),
```

Comment:
We can also add Japanese, Malayalam, Serbian, Hungarian, and Korean (all have at least 50 strings translated on transifex).